.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,27 +1,60 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
+title: "🐛 "
 labels: bug
-assignees: ''
-
+assignees: ""
```

Comment:
Is there a significance between single and double quotes?

Reply:
no

Reply:
What's the reason for the change, then?